docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1267 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.examples</groupId>
+    <artifactId>helidon-mp-tutorial</artifactId>  <1>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <helidon.version>{helidon-version}</helidon.version>
+        <mainClass>io.helidon.examples.Main</mainClass>
+        <maven.compiler.source>8</maven.compiler.source> <2>
+        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.helidon</groupId>
+                <artifactId>helidon-bom</artifactId> <3>
+                <version>${helidon.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.bundles</groupId>
+            <artifactId>helidon-microprofile-2.2</artifactId> <4>
+        </dependency>
+    </dependencies>
+
+</project>
+----
+
+This POM files contains just the basics that we need to get started.  
+
+<1> It establishes the Maven coordinates for the new project.
+<2> The `properties` section tells Maven to use Java 8 for the source
+ and target versions, and sets the `mainClass` which will be used later
+ when building a JAR file.  The class will be created later in the
+ tutorial.
+<3> It includes the Helidon BOM POM  (in the `dependencyManagement` section).
+<4> It adds a dependency for the MicroProfile 2.2 bundle which allows the use of
+ MicroProfile 2.2 features in the application.
+
+TIP: MicroProfile 2.2 contains features like Metrics, Health Check,
+ Streams Operators, Open Tracing, OpenAPI, REST client, and fault
+ tolerance. You can find detailed information about MicroProfile 2.2 on the
+ https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-2.2[Eclipse MicroProfile 2.2] site.
+
+With this POM file, the application can be built successfully with Maven:
+
+[source,bash]
+----
+mvn clean package
+----
+
+This will create a JAR file in the `target` directory.
+
+TIP: The warning message `JAR will be empty - no content was marked for inclusion!` 
+ can be ignored for now  because there is no actual content in the 
+ application yet.
+
+== Create the MicroProfile application
+
+The actual application logic can be created now.
+ The first step is to add the application class. Create a directory
+ for your source code, and then create directories for the package 
+ hierarchy:
+
+[source,bash]
+.Create directories for source code
+----
+mkdir -p src/main/java/io/helidon/examples
```

Comment:
Note that the project's `artifactId` says `-tutorial`; maybe `examples` should be `tutorial`?

Reply:
I think it is ok, it is an example after all, just one that also happens to be a tutorial :) But if you feel strongly about it, I will go through and change it everywhere.